.vscode/extensions.json

```diff
@@ -1,9 +1,7 @@
 {
   "recommendations": [
+    "charliermarsh.ruff",
```

Comment:
@eliotwrobson In your experience, how does the Ruff extension work for you compared to Flake8? Similar experience?

Reply:
Much better, it runs super fast and there's no reason to fuss with any plugins. I habitually switch to it on all my projects because of this.

Reply:
@eliotwrobson Honestly, I was considering switching my projects to use Ruff as well. it sounds pretty great, and I like the idea of consolidating my dev dependencies.

Just to do my due diligence, I'm going to install the VS Code extension under this PR to give it a try before we merge.

Reply:
It's honestly a great experince to use so I think it's worth the time 😄

Reply:
@eliotwrobson Wow, that is blazing fast. So fast that I'm getting syntax errors from Ruff before I can finish typing out a new line of code 🤣. But that's fine! I don't think that will get in the way of development.

With that, I think this PR is good to go, and I will go ahead and merge.